docs/docs/platforms/openshift/OPENSHIFT_FAQ.md

```diff
@@ -0,0 +1,112 @@
+
+:octicons-thumbsup-16: Running with a Restricted SCC is the recommended way to deploy Atlassian DC Helm charts. See: [Restricted SCC](../OPENSHIFT/#__tabbed_1_1)
+
+
+## NFS Permission Fixer Is Disabled, How Can I Make Shared Home Writable
+
+When running Atlassian DC containers with a restricted or nonroot SCC, it is not possible to run nfs-permission fixer container because it runs as root. You need to make sure the volume is writable either for an unprivileged restricted SCC user or a user defined in `securityContext` (with UID like 2001, 2002, 2003 depending on the product).
+
+If you run NFS server, you may either change permissions directly on the NFS server instance, or create a temporary ServiceAccount, grant it anyuid SCC privileges, create a pod that uses this ServiceAccount, mounts shared-home volume and recursively changes owner of the shared home directory. It can take longer for larger volumes. 
+
+## Can I Use a Custom Certificate and Key for My Route?
+
+By default all Routes that the Helm charts will create will use OpenShift Router default certificate and key. If you want to use a custom crt-key pair for an individual Route, create a [TLS secret](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret_tls/){.external} with `tls.crt`, `tls.key` and `ca.crt` (optional) in `data` and provide the secret name in Helm values:
+
+```
+ingress:
+  create: true
+  openShiftRoute: true
+  tlsSecretName: my-tls-secret
+```
+
+## How Can I Configure OpenShift Routes?
+
+You can provide custom annotations for your Routes in the `ingress` Helm values stanza. For example, to increase connection timeout, annotate your route:
+
+```yaml
+ingress:
+  create: true
+  openShiftRoute: true
+  annotations:
+    haproxy.router.openshift.io/timeout: 60s
+```
+
+You will find a list of available Route specific annotations in the [official documentation](https://docs.openshift.com/container-platform/4.14/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration){.external}.
+
+[Route HTTP Headers](https://docs.openshift.com/container-platform/4.14/rest_api/network_apis/route-route-openshift-io-v1.html#spec-httpheaders){.external} can be configured in the `ingress` Helm values stanza:
+
+```yaml
+ingress:
+  create: true
+  openShiftRoute: true
+  routeHttpHeaders:
+    <your-actions>
+```
+
+## What Rote TLS Termination Types Are Supported
+
+Atlassian DC Helm charts only support **Edge TLS termination**, i.e. TLS termination is performed by the OpenShift router. Passthrough and Reencrypt TLS termination isn't currently supported. See: [Secured Routes](https://docs.openshift.com/container-platform/4.14/networking/routes/secured-routes.html){.external}
```

Comment:
I'd start the "Passthrough and Reencrypt TLS..." sentence on a new line, just to make the split between what's supported and what's not a bit clearer.